src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -492,9 +513,13 @@ ucs_status_t uct_dc_mlx5_ep_get_bcopy(uct_ep_h tl_ep,
     UCT_RC_IFACE_GET_TX_GET_BCOPY_DESC(&iface->super.super,
                                        &iface->super.super.tx.mp,
                                        desc, unpack_cb, comp, arg, length);
+    UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
+    fm_ce_se |= uct_rc_ep_fm(&iface->super.super, &txwq->fi,
+                             iface->super.config.atomic_fence_flag);
```

Comment:
is this because for GET operations we still use the default rkey?
maybe wrap this in common function similar to uct_rc_mlx5_ep_fence_put(): uct_rc_mlx5_ep_fence_get()?